i18n/ru.json

```diff
@@ -1,5 +1,5 @@
 {
-    "PrivateBin": "PrivateBin",
+    "PrivateBin": "Приватный Бин",
```

Comment:
Hmm...

Reply:
I guess you should comment this on Crowdin...

Reply:
Already done ;)